main.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <!-- create a navbar as a header  -->
+    <header>
+        <nav class="nav">
+            <ul class="ul ul1">
+                <li class="navli nav-li1">My logo</li>
+                <li class="navli nav-li2">Portfolio</li>
+                <li class="navli nav-li3">About</li>
+                <li class="navli nav-li4">Contact</li>
+            </ul>
```

Comment:
- Hey, please, kindly hide the links as they're not supposed to be visible in the mobile version.
- Also, kindly add the missing hamburger as seen in the design template to make your design look closely like the mobile version. see the images below to compare your design:
- The My logo should be a link to the `index.html` to ensure navigation from the home page to other pages in the website.

*design template*
![Screenshot from 2023-01-12 09-12-52](https://user-images.githubusercontent.com/59397159/212013083-8b11ed37-c36a-48c1-9128-265147278dec.png)

*your design*
![Screenshot from 2023-01-12 09-13-38](https://user-images.githubusercontent.com/59397159/212013313-6cdc977b-1905-485d-8ae0-fc9927341301.png)